packages/neon/neon/lib/src/pages/settings.dart

```diff
@@ -181,26 +175,15 @@ class _SettingsPageState extends State<SettingsPage> {
                         for (final account in accountsSnapshot.data!) ...[
                           NeonAccountSettingsTile(
                             account: account,
-                            onTap: () async {
-                              await Navigator.of(context).push(
-                                MaterialPageRoute(
-                                  builder: (final context) => AccountSettingsPage(
-                                    bloc: accountsBloc,
-                                    account: account,
-                                  ),
-                                ),
-                              );
+                            onTap: () {
+                              AccountSettingsRoute(accountid: account.id).go(context);
                             },
                           ),
                         ],
                         CustomSettingsTile(
                           title: ElevatedButton.icon(
-                            onPressed: () async {
-                              await Navigator.of(context).push(
-                                MaterialPageRoute(
-                                  builder: (final context) => const LoginPage(),
-                                ),
-                              );
+                            onPressed: () {
+                              const LoginRoute().go(context);
```

Comment:
This doesn't work. It just goes back to the home page and the login page doesn't show up.